### ISSUE TITLE
Option to display alt text with the image

### ISSUE THREAD

**Issue by cloudrac3r** [cloudrac3r/bibliogram#49]
This request is from "GNUser", sent to me by email.

There should be an option to display the alt text as well as the image.

This depends on #28.

**Comment by cloudrac3r** [comment]
No longer waiting. A control exists for this, but it's not enabled or implemented.
